mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1758,6 +1786,16 @@
 	<!-- Dialog button text for canceling the rename top site prompt. -->
 	<string name="top_sites_rename_dialog_cancel">Cancel</string>
 
+    <!-- In-activate tabs in the tabs tray -->
+    <!-- Title text displayed in the tabs tray when a tab has been unused for 4 days. -->
+    <string name="inactive_tabs_title">Inactive tabs</string>
+    <!-- A description below the section of "inactive" tabs to notify the user when those tabs will be closed, if appropriate. -->
```

Comment:
Nit: Just to be extra clear with localizers, please explain in comments what the placeholder will be replaced with

Reply:
Thank you! I think this can work for this.
Some context: some of our localizers don't really understand looking at other string comments given how Pontoon UX is made, and their actual knowledge on how strings work.
But given how you explained it, I think it should be ok :)